<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QWidget" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>950</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>950</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>950</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="OpenButton">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>210</y>
     <width>750</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateButton">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>410</y>
     <width>750</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TextButton">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>610</y>
     <width>750</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Найти в тексте</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
